src/sass/components/_grid.scss

```diff
@@ -112,7 +110,7 @@ $gutter: $md;
   }
 
   &--tight > [class^="#{$prefix}-cols"] {
-    padding: 0 ($gutter/2);
+    padding: 0 ($gutter);
   }
```

Comment:
See previous comment. This will need to be the positive version of the negative number you'll need to divide by in the `rvt-row--tight` modifier above.